middleware/middleware-1.3.14.md

```diff
@@ -0,0 +1,80 @@
+
+## Changed behavior: DSFinV-K receipt referencing
+We've updated our DSFinV-K generation logic to now include a broader variety of different receipt connections into the allocation groups (i.e. "Abrechnungskreis") part of the export. This file now includes one entry for each of the following connections:
+- `cbPreviousReceiptReference`
+- `cbReceiptReference`
+- `cbArea`
+
+This means that no matter which approach our partners use to connect receipts, the connections will now be made visible in the export. This behavior is backwards-compatible and generates correct exports for previously transmitted receipts as well.
+
+## Stability improvement: DSFinV-K export with old data
+While introducing the previous change, we generally improved the stability of our DSFinV-K export to be more compatible with older queues. This should fix several rare issues users were experiencing when querying exports (e.g. missing files).
+
+## Bug fix: Wrong item delimiter sent to TSE for info-order receipts
+When signing info-orders (which are treated as `Bestellung-V1` on the TSE) with more than one ChargeItem, the Middleware used a wrong item delimiter when generating the TSE payload - instead of `\r`, `\n` was used. As this is only a formal issue, the content of the data was still correctly signed.
+
+This is now resolved, and the correct delimiter is used. Generating the DSFinV-K is not affected by this change.
+
+## Bug fix: It was technically possible to operate a stopped Queue
+Due to an issue in the Middleware, it was technically still possible to send receipts to a stopped Queue. However, since the TSE is usually deactivated in this case (if the respective flag is not used), this could lead to exceptions.
+
+With this fix, the Queue now correctly responds with the _ftState_ `0x4445000000000001` after it was deactivated, and doesn't contact the SCU and therefore the TSE anymore.
+
+## Bug fix: Launcher sometimes fails to start when operations take too long
+Due to a default limitation in Windows services, services must not take more than 30 seconds to start, otherwise they'll be flagged as "not responding" by Windows. This issue infrequently happens with components that need to download 3rd party components during the first initialization (e.g. Swissbit Cloud/Deutsche Fiskal), which may take more than 30 seconds with slow internet connections.
+
+This issue is now resolved, as the Launcher automatically requests more startup time starting from this version.
+
+## Bug fix: Diebold Nixdorf SCU throws exception during self test
+Due to unexpected behavior of Diebold Nixdorf TSEs with older firmware versions, the Middleware could run into issues when querying the TSE's current state details. As this operation is executed frequently during the self test of the TSE, this could lead to recurrent issues.
+
+The Middleware was updated to be compatible with the respective firmware versions.
```

Comment:
Should we add information on the firmware version since this is printing a warning in the log